script/nodenv-install.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Install our selected Node version defined in the .node-version file.
+#
+# If our node-build version is outdated and it can't build the version we want
+# then we try upgrading node-build and installing again.
+
+if nodenv install --skip-existing; then
+  echo "Correct Node version is installed."
+else
+  echo "Upgrading node-build:"
+
+  if command -v brew &> /dev/null; then
```

Comment:
I didn't know about that `command`, which is pretty convenient here. 
I used to use `if [[ "`uname`" == "Darwin" ]]; then` to test if current is Mac:

https://github.com/openfoodfoundation/openfoodnetwork/blob/ace74dc42d8855c908c7ce7bfb8056609c466718/script/rspec-slow-repeat#L33-L37